tests/unit/multi_table/test_base.py

```diff
@@ -40,7 +40,9 @@ def test__initialize_models(self):
         locales = ['en_CA', 'fr_CA']
         instance = Mock()
         instance._table_synthesizers = {}
-        instance._table_parameters = {'nesreca': {'default_distribution': 'gamma'}}
+        instance._table_parameters = {
+            'nesreca': {'default_distribution': 'gamma', 'locales': ['fr_CA', 'en_CA']},
```

Comment:
Use previous variable:
```suggestion
            'nesreca': {'default_distribution': 'gamma', 'locales': locales[::-1]},
```

Reply:
For completeness I'll just use a completely different locale list 😅